VariantSurvival_package/R/VariantSurvival.R

```diff
@@ -1,369 +1,740 @@
-                                  ),
-                                  selectInput(
-                                    inputId = "n_svs_max",
-                                    label = "max",
-                                    choices = NULL,
-                                    selected = FALSE
-                                  ),
-                                  checkboxGroupInput(
-                                    "km_feat",
-                                    "Plot layout options:",
-                                    choices = c(
-                                      "confidence interval" = "conf_itv",
-                                      "risk table" = "risk_table",
-                                      "y grid line" = "grid_line"
-                                    )
-                                  ),
-                                  circle = TRUE,
-                                  status = "danger",
-                                  icon = icon("gear"),
-                                  width = "300px",
-                                  tooltip = tooltipOptions(title = "Click to see inputs !")
-                                ),
-                                shinycssloaders::withSpinner(plotOutput(outputId = "plot_km", width = "100%"))
-                              )
-                            ),
-                            fluidRow(
-                              column(
-                                12,
-                                div(style = "height:200px; Bottomleft"),
-                                checkboxInput("life_table_multiple_model",
-                                              "Display life table",
-                                              value = FALSE),
-                                tabBox(
-                                  id = "myBox_mm",
-                                  title = "",
-                                  width = 12,
-                                  tabPanel("with SV",
-                                           span(DT::dataTableOutput("lt_mm_0"))),
-                                  tabPanel("without SV",
-                                           span(DT::dataTableOutput("lt_mm_1")))
-                                )
-                              )
-                            )),
-                   tabPanel("Null model",
-                            fluidRow(
-                              column(
-                                12,
-                                div(style = "height:70px; Topleft"),
-                                shinycssloaders::withSpinner(plotOutput(outputId = "null_model_km"))
+        ),
+      shiny::tabPanel(
+        "Kaplan-Meier",
+        shiny::fluidRow(
+          shiny::column(
+            width = 12,
+            shinydashboard::tabBox(
+              width = "100%",
+              id = "tabset_km",
+              shiny::tabPanel(
+                title = span("Multiple Model", 
```

Comment:
span() requires scope (found multiple ones for function span, don't know which one is required here)